wrappers/fastp/meta.yaml

```diff
@@ -0,0 +1,18 @@
+name: "fastp"
```

Comment:
finalement the meta.yaml est peut etre pas necessaire ici, c'est la doc de fastp par contr on pourrait avoir un meta.yaml uniforme pour sequana-warappers a reflechir et définir qui pourrait etre utiliser dans un future doc. Je dirais qu'on s'embete par pour le moment avec les meta.yaml peut etre ?

Reply:
J'étais plus team on fait des readme en markdown consultable directement sur le github

Reply:
OK, je supprime le yaml et ça me va de voir après avec des markdowm

Reply:
👌